cli.go

```diff
@@ -45,6 +45,11 @@ type Cli struct {
 	Verbose    bool
 }
 
+type statementWithFlag struct {
```

Comment:
Can we consider another name as we already have many structs using `statement` in the name?

Some examples: `batchExecution`, `executionUnit`, `command`.

Reply:
I would like to choose `executionUnit` or `command`.
`command` is simpler.

Reply:
Fixed in https://github.com/cloudspannerecosystem/spanner-cli/pull/40/commits/a4b4106f5494fd2caceb36cfda5a7b2fe0b8bd01